data/templates/default.context

```diff
@@ -104,6 +104,12 @@ $endif$
 \setupxtable[body][]
 \setupxtable[foot][bottomframe=on]
 
+$if(includesource)$
+$for(sourcefile)$
+\attachment[file=$curdir$/$sourcefile$,method=hidden]
+$endfor$
+$endif$
+
```

Comment:
Remove line 112, as it creates an extra blank line in context documents even if the new variable is not set (that's why the tests fail).  If you want the blank line in the generated context, put it before the endif, and then it will only be included if the attachment command is.

Reply:
Done.